visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -323,16 +323,17 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
                     this.props.accuracyPickerProps.selectedAccuracyKey);
             });
             const disparityMetric = this.state.disparityInOutcomes ?
-                (this.props.dashboardContext.modelMetadata.predictionType === PredictionTypes.binaryClassification ?
-                    "selection_rate" : "average") :
+            (this.props.dashboardContext.modelMetadata.predictionType === PredictionTypes.binaryClassification ?
+                this.props.parityPickerProps.selectedParityKey : "average") :
                 this.props.accuracyPickerProps.selectedAccuracyKey;
+            const parityMode = this.props.parityPickerProps.parityOptions.filter(option => option.key == this.props.parityPickerProps.selectedParityKey)[0].parityModes[0];
```

Comment:
not sure where the list here is being set, but it may be easier to have a pairityMode control in this view to set the pairityMode, or if the only one you want to support ratio is selectio_rate, then the options present in the UI are selectionRateRatio, selectionRateDiff, ...., each with a unique key and then have a switch statement on the key and set disparity metric and parity mode based on the selected key in the single ui radio button list

Reply:
That makes sense as a way to multiplex the selection of disparity metric as well as the parity mode simultaneously in the wizard.